msi/release.sh

```diff
@@ -62,6 +147,8 @@ BUILD_EXTRA_WINPATH="$(cd "$SCRIPT_PATH"/.. && pwd -W | tr / \\\\)"
             </Component>
 EOF
 echo "$LIST" |
+sort |
+uniq |
```

Comment:
@dscho: Yes, `make-file-list.sh` is sorted, but by package name, since each line is in the format `<package name> <filepath>`. This sort that takes place here sorts by directory/filename (since by this point, the package name has been stripped off.. Now, is this _needed_? No, but it was really helpful to quickly find a file during development. I'll remove it--unless you want to keep it--it is extra overhead.